lib/rubyhub/config/templates/.frontend_description.txt

```diff
@@ -0,0 +1,9 @@
+### Before submitting the merge request make sure the following are checked
```

Comment:
it wouldn't be default template

Consider that it might have(or not) template OR could have description